mobile/app/src/main/java/org/exarhteam/iitc_mobile/prefs/PluginPreferenceActivity.java

```diff
@@ -140,11 +141,16 @@ public boolean onOptionsItemSelected(final MenuItem item) {
             case R.id.menu_plugins_add:
                 if (mFileManager.checkWriteStoragePermissionGranted()) {
                     // create the chooser Intent
+
+                    Log.i("ABLAGE2--> "+ Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOCUMENTS) + "/IITC_Mobile/");
```

Comment:
The line that's not needed

Reply:
For testing i need it.. because i cannot write to the old path to store the external user plugins..

Will also add a TODO here for removing the LOG